lib/util/git/index.ts

```diff
@@ -900,7 +903,30 @@ export async function mergeBranch(branchName: string): Promise<void> {
       ]),
     );
     status = await git.status();
-    await gitRetry(() => git.merge(['--ff-only', branchName]));
+    if (
+      mergeStrategy === 'fast-forward' ||
+      mergeStrategy === 'rebase' ||
```

Comment:
Should we do a --ff-only merge for rebase? Or instead:
- do a rebase merge, or
- warn that rebase is unsupported?

Reply:
Also needs docs

Reply:
Isn't a rebase essentially the same as a fast-forward when a fast-forward is possible? The git tree appears identical, right? :thinking: I'm asking because if only fast-forward is implemented right now, I assumed that the branches are always in a state where they can be fast-forwarded. So there's no need to implement `rebase`. Or am I wrong?

Reply:
Yes, I think you're right. "rebase merge" makes sense because github offers it for PRs, but there is no such things as e.g. `git --rebase merge` and if you do a rebase then merge then it's the same as a FF.